pkg/handlers/authorization/handler.go

```diff
@@ -56,10 +58,14 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 		return req, fmt.Errorf("could not run SAR or fech projects: %v", err)
 	}
 	context.UserName = rolesProjects.review.UserName()
-	context.Projects = rolesProjects.projects
 	if rolesProjects.review.UserName() != "" {
 		req.Header.Set(headerForwardedUser, context.UserName)
 	}
+	context.Projects = rolesProjects.projects
+	for _, project := range context.Projects {
+		req.Header.Add(headerForwardedNamespace, project.Name)
```

Comment:
for the sake of building our DLS do we need this in another format?

Reply:
> for the sake of building our DLS do we need this in another format?
Without a change to OD and possibly how it does parameter substitution, the api for attributes only takes a key value pair.  On the other end via my proxy changes, the only way to handle this is to take in a single value that is a list or send multiple keys which get assembled into a list.